lib/validate.js

```diff
@@ -39,6 +59,22 @@ module.exports = function validateMiddleware (opts) {
     }
   });
 
+  assert(
+    isObject(opts.joiValidateOptions),
```

Comment:
Ah I see. You've set it to an Object on line #36 so if it's not provided this check will pass.